npm/src/ux/utils.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function formatThousandSepF64(val: number): string {
```

Comment:
Not sure we need either of these methods. The number formatter with options such as 'maximumFractionDigits', and includes locale-specific separators by default, has been widely implemented for about 10 years across browsers.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat